blindml/runner.py

```diff
@@ -5,24 +5,16 @@
 
 def run(task_file_fp):
     task = parse_task_capsule(task_file_fp)
-    task.search_for_model()
-    while not task.get_model_search_update():
-        print("no model trained yet")
-        time.sleep(5)
+    model = task_auto_sk_model
```

Comment:
`task_auto_sk_model` is not defined in this function (nor in this file...)

Reply:
missing . fixed in 7c840fa9ee580e883d7d30a5758ac415c24fc8be